install_process.go

```diff
@@ -69,11 +73,11 @@ func (p PipInstallProcess) Execute(workingDir, targetPath, cachePath string) err
 	return nil
 }
 
-func onlineArgs(cachePath string) []string {
+func onlineArgs(cachePath string, requirementPath string) []string {
 	return []string{
 		"install",
 		"--requirement",
-		"requirements.txt",
+		requirementPath,
```

Comment:
Since the buildpack need to install packages based on a file (instead of providing the package name directly to the CLI), I think it's better to be explicit here (instead of "silently" use the `PIP_REQUIREMENT` env).

Reply:
That being said, should we also provide `BP_PIP_REQUIREMENT` (and support both of them) to emphasize this behavior?